### PR TITLE
Fix "method_lines" issue in CreateUserController.php

### PR DESCRIPTION
**Pull request by AbderrahimSoubaiElidrissi**

### CODE REVIEW

File: src/Controllers/Users/CreateUserController.php [hunk 1]
```diff
@@ -32,8 +32,7 @@ public function __invoke(ServerRequestInterface $request, array $args): array
             ]);
 
             /** @var Redis $redis */
-            $redis = $this->container->get(Redis::class);
-            $result = $redis->lPush("email:queue",
+            $result = $this->container->get(Redis::class)->lPush("email:queue",
```
Comment (santiagolizardo) [inline src/Controllers/Users/CreateUserController.php:35]: The comment (var Redis) should have been removed as well as the result variable not longer holds a Redis instance.

**Review by santiagolizardo — CHANGES_REQUESTED** [review]
Thanks for your PR @AbderrahimSoubaiElidrissi . Unfortunately the change you are suggesting is not a great improvement over existing code.
If you want to make a difference to this class why don't you extract all the code that make the push to Redis to a new method on the same class to have two smaller methods instead of a single big one?
Thanks!

**Comment by santiagolizardo** [comment]
Closing due to the inactivity

**Comment by wr4** [comment]
This type of spam people making PRs just to win a t shirt, thanks to DO update this PRs like this will not be counted to this dude, "Talk is cheap. Show me the code".

**Pull request closed without merging**